perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -377,7 +392,19 @@ def RunNetperf(vm, benchmark_name, server_ip, num_streams):
                      server_ip=server_ip,
                      length=FLAGS.netperf_test_length,
                      output_selector=OUTPUT_SELECTOR,
-                     confidence=confidence, verbosity=verbosity)
+                     confidence=confidence,
+                     verbosity=verbosity)
+
+  if benchmark_name.upper() == 'UDP_STREAM':
+    netperf_cmd += (' -R 1 -m {send_size} -M {send_size} '.format(
+                    send_size=FLAGS.netperf_udp_stream_send_size_in_bytes))
+    metadata['netperf_send_size_in_bytes'] = FLAGS.netperf_udp_stream_send_size_in_bytes
+
+  elif benchmark_name.upper() == 'TCP_STREAM':
+    netperf_cmd += (' -R 1 -m {send_size} -M {send_size} '.format(
```

Comment:
Per rick's comment it seems that we can drop the `-m` and `-M` here.

As well as the `-R`